monitoring/gosh-mon/crons/cron.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
I believe it would be better to rename this cron into something meaningful. Even gosh-cron is better than this